toolchain/parse/typed_nodes_test.cpp

```diff
@@ -252,18 +253,16 @@ TEST_F(TypedNodeTest, VerifyExtractTraceClassDecl) {
   EXPECT_THAT(err.message(), testing::MatchesRegex(
                                  R"Trace(Aggregate [^:]*: begin
 Aggregate [^:]*: begin
-Aggregate [^:]*: begin
```

Comment:
What makes this go away? (Not really sure what to make of these tests yet..)

Reply:
I think before, `NameAndParams` existed as a member aggregate. It's gone without a replacement.